sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_sync_token.py

```diff
@@ -30,15 +30,19 @@
 
 
 class SyncToken(object):
```

Comment:
Do we still need to inherit from object?

Reply:
I think it's safe to get rid of.